DOCUMENTATION.md

```diff
@@ -97,61 +129,48 @@ include runner_make osm_make
 
 * Create a [channel](https://slack.com/help/articles/201402297-Create-a-channel);
 * Generate a Slack App and [configure it](https://github.com/kasunkv/slack-notification/blob/master/generate-slack-token.md);
 * Add it to your [channel](https://slack.com/help/articles/202035138-Add-apps-to-your-Slack-workspace);
 * [Get the Bot User OAuth Token](https://github.com/kasunkv/slack-notification/blob/master/generate-slack-token.md#4-copy-oauth-access-token--use-in-azure-pipelines) n e.g. 'xoxb-111-222-xxxxx'. Bot OAuth Token will be stored in the SLACK_KEY variable in the file config.inc.sh. The angle brackets around your_key are in need to be removed.
 
-4. Copy [config.inc.sh.sample](config.inc.sh.sample) from geocint-runner to ~/:
-```shell
-cp ~/geocint-runner/config.inc.sh.sample ~/config.inc.sh
-```
-Open ~/config.inc.sh and set the necessary values for variables. See comments at this file for details.
-
-5. Run installers:
+4. Copy [config.inc.sh.sample](config.inc.sh.sample) from geocint-runner to working directory and name config.inc.sh.
 
-- ~/geocint-runner/runner-install.sh (necessary dependencies to run a runner part)
-- ~/geocint-openstreetmap/openstreetmap-install.sh (necessary dependencies to run a runner part)
-- ~/[geocint-private]/install.sh (Do not forget to install any of your custom dependancies, if any. Change [geocint-private] to the name of your private repository. )
-- ~/geocint-runner/scripts/set_modes.sh (create /public_html and /domlogs folders and set access modes)
+Then open config.inc.sh and set the necessary values for variables. See comments at this file for details.
 
-Please also note, that installers are run automatically, when the pipeline is started via start_geocint.sh
-
-6. Create PostgreSQL role and create PostgreSQL extensions (replace "gis" with your username if you have different). Follow the next steps below :
+5. Run installation:
 ```shell
-	# open psql console by admin (user postgres)
-	sudo -u postgres psql
-	# create role and database
-	create role gis login;
-	create database gis owner gis;
-	# connect to database
-	\c gis
-	# create extensions
-	create extension postgis;
-	create extension postgis_raster;
-	create extension postgis_sfcgal;
-	create extension postgis_topology;
-	create extension h3;
-	create extension h3_postgis;
-	# you can create any additional extension, that you need
-	# quit psql console
-	\q
+	# install cmake if not exists
+	sudo apt install -y cmake
+	# move to the folder with installation Makefile
+	cd /your/working/directory/geocint-runner/scripts/
+	# run installation (you should set path to config.inc.sh file)
+	make install configuration_file=/your/working/directory/config.inc.sh
+	# reload ~/.bashrc file
+	source ${HOME}/.bashrc
 ```
-7. Set the crontab to autostart the pipeline. 
+
+6. Set the crontab to autostart the pipeline. 
```

Comment:
We need mkdir -p geocint for cases, when geocint folder wasn't created or were removed.
Because without it logs redirection "> /home/gis/geocint/log.txt" will drop an error.